allennlp/interpret/saliency_interpreters/simple_gradient.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=protected-access
+from typing import List
+import math
+import numpy
+from allennlp.common.util import JsonDict, sanitize
+from allennlp.modules.text_field_embedders import TextFieldEmbedder
+from allennlp.interpret.saliency_interpreters import SaliencyInterpreter
+
+@SaliencyInterpreter.register('simple-gradients-interpreter')
+class SimpleGradient(SaliencyInterpreter):
+    def saliency_interpret_from_json(self, inputs: JsonDict) -> JsonDict:
+        """
+        Interprets the model's prediction for inputs.
+        Gets the gradients of the loss with respect to the input
+        and returns those gradients normalized and sanitized.
+        """
+        # Convert inputs to labeled instances
```

Comment:
This comment duplicates almost exactly the name of the method below it and adds no new information.  It's unnecessary.

Reply:
Just remove the comment.  See [here](https://google.github.io/styleguide/pyguide.html#385-block-and-inline-comments) and the "don'ts" section [here](https://google.github.io/styleguide/cppguide.html#Function_Argument_Comments).